### PR TITLE
internal/namespace: make default namespace secure by default

### PR DESCRIPTION
**Pull request by ben-toogood**

### CODE REVIEW

**Comment by ben-toogood** [comment]
All credit to @crufter, this is an adapted version of his PR: https://github.com/micro/micro/pull/1024

File: client/cli/token/token.go [hunk 1]
```diff
@@ -57,3 +57,18 @@ func Save(envName string, token *auth.Token) error {
...
 }
+
+// Remove deletes a token. Useful when trying to reset test
+// for example at testing: not having a token is a different state
+// than having an invalid token.
+func Remove(envName string) error {
+	if err := config.Set("", "micro", "auth", envName, "token"); err != nil {
+		return err
+	}
+	// Store the refresh token in micro config
```
Comment (domwong) [inline client/cli/token/token.go:68]: duff comment

File: client/cli/token/token.go [hunk 2]
```diff
@@ -57,3 +57,18 @@ func Save(envName string, token *auth.Token) error {
...
+	if err := config.Set("", "micro", "auth", envName, "refresh-token"); err != nil {
+		return err
+	}
+	// Store the refresh token in micro config
```
Comment (domwong) [inline client/cli/token/token.go:72]: duff comment

File: internal/config/config.go [hunk 3]
```diff
@@ -22,7 +22,8 @@ var config = newConfig()
 
 // Get a value from the .micro file
 func Get(path ...string) (string, error) {
-	tk := config.Get(path...).String("")
+	c := newConfig()
```
Comment (domwong) [inline internal/config/config.go:25]: sheesh. I'll take a look at the config watcher code

**Pull request merged**